RLTest/__main__.py

```diff
@@ -354,7 +354,8 @@ def takeEnvDown(self, fullShutDown=False):
                                    env=self.currEnv)
 
         if needShutdown:
-            self.currEnv.flush()
+            if self.currEnv.isUp():
+                self.currEnv.flush()
```

Comment:
If we are going to stop the env anyway why the flush is needed?

Reply:
I don't know why we flush the env beforehand? Does Redis automatically flush on exit? Otherwise, it might be a good way to check for leaks?

Reply:
Ohh right, leaks is a good reason.